lib/modules/manager/gradle/artifacts.ts

```diff
@@ -86,6 +89,34 @@ async function getGradleVersion(gradlewFile: string): Promise<string | null> {
   return extractResult ? extractResult.version : null;
 }
 
+async function buildUpdateVerificationMetadataCmd(
+  verificationMetadataFile: string | undefined,
+  baseCmd: string
+): Promise<string | undefined> {
+  if (verificationMetadataFile) {
+    const hashTypes: string[] = [];
+    const verificationMetadata = await readLocalFile(verificationMetadataFile);
+    for (const hashType of ['sha256', 'sha512']) {
+      if (verificationMetadata?.includes(`<${hashType}`)) {
+        hashTypes.push(hashType);
+      }
+    }
+    if (!hashTypes.length) {
+      hashTypes.push('sha256');
+    }
+    if (
```

Comment:
Since metadata verification can also be toggled, wouldn't it make sense to also do a similar check for `<verify-metadata>true</verify-metadata>`? Adding hash algorithms only makes sense if this condition is fulfilled.

Reply:
good idea